test/integration/smoke/test_service_offerings.py

```diff
@@ -811,8 +817,28 @@ def test_06_disk_offering_strictness_false(self):
             "Check service offering of the VM"
         )
 
+        if fullClone[0].value.lower() == "false":
+            self.updateVmwareSettings(True)
+
         return
 
+    def updateVmwareSettings(self, tearDown):
+        value = "false"
+        if not tearDown:
+            value = "true"
+        if self.hypervisor.lower() == 'vmware':
+            Configurations.update(self.apiclient,
+                                  "vmware.create.full.clone",
+                                  value)
+            allStoragePools = StoragePool.list(
+                self.apiclient
+            )
+            for pool in allStoragePools:
```

Comment:
Do we need to update all the storage pools? If that is the case I think we can keep the initial values per storage pool in memory and once the test finishes, we can revert them back to the initial value

Reply:
I am doing that at the moment @nvazquez . I'll update the PR soon